lisa/operating_system.py

```diff
@@ -372,7 +373,16 @@ def _package_exists(self, package: str, signed: bool = True) -> bool:
 
     def _get_os_version(self) -> OsVersion:
         os_version = OsVersion("")
-        cmd_result = self._node.execute(cmd="lsb_release -as", no_error_log=True)
+        # Output of 'lsb_release -as' doesn't match the pattern.
```

Comment:
This comments explained the history of change, it should be in commit message, not in code comments. The code comments should show examples of outputs, which distro it's from, and which one is tricky.